src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntryAsync.File.Tests.Unix.cs

```diff
@@ -149,5 +149,49 @@ await using (TarReader reader = new TarReader(archive))
                 }
             }, format.ToString(), new RemoteInvokeOptions { RunAsSudo = true }).Dispose();
         }
+        
+        [ConditionalTheory(nameof(IsRemoteExecutorSupportedAndPrivilegedProcess))]
```

Comment:
You have renamed to `IsRemoteExecutorSupportedAndPrivilegedProcess ` to `IsRemoteExecutorSupportedAndPrivilegedProcess` but not included the new property in `TarTestsBase.cs`

Reply:
Thanks. That property name change had not been backported yet. I included a commit to rename it, as well as all the callsites.

Reply:
Thanks. That property name change had not been backported yet. I included a commit to rename it, as well as all the callsites.